src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/health/BankHolidayHealth.kt

```diff
@@ -7,5 +7,6 @@ import org.springframework.web.reactive.function.client.WebClient
 @Component("bankHoliday")
 class BankHolidayHealth(
   webClient: WebClient,
+  @Value("bankHoliday") componentName: String,
```

Comment:
Is there a reason to have the component name dynamic here but static in other places? Be nice to be consistent? Its a shame we cant reuse the `@Component` name

Reply:
Can you use the name from `@Component` - genuine question as that would be much better... 😄 

This is the first bit o' Kotlin/Java I've done in well over a decade so there's lots about stuff I don't know

Reply:
Lets merge like this, I'll have a look around but initial query suggests its not an option